examples/jsm/renderers/common/RenderObjects.js

```diff
@@ -17,10 +17,10 @@ class RenderObjects {
 
 	}
 
-	get( object, material, scene, camera, lightsNode, passId ) {
+	get( object, material, scene, camera, lightsNode, passId, renderContext ) {
```

Comment:
I think better this signature: `get( object, material, scene, camera, lightsNode, renderContext, passId )`
`passId` must be opcional.